src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    private Ball ball = new Ball();
```

Comment:
this variable should be local 
because you are creating a new ball each time method `getRandomBall()` is invoked